lib/blog.js

```diff
@@ -15,19 +16,20 @@ function parseMarkdown (markdown) {
 
 const POSTS_PATH = path.join(__dirname, '..', 'data', 'blog')
 class BlogPost {
-  static async getAll () {
+  static async getAll (language) {
     const files = await fs.readdir(POSTS_PATH)
     const slugs = files.map(file => path.basename(file, '.md'))
-    return slugs.map(BlogPost.get)
+    return slugs.map((slug) => BlogPost.get(slug, language))
   }
 
-  static get (slug) {
-    return new BlogPost(slug)
+  static get (slug, language) {
```

Comment:
`BlogPost.get` doesn't seem to add any functionality beyond `new BlogPost`. Is this just here to avoid using the `new` keyword?

Reply:
This is refactored not by me, I'm not able to say the real surface beyond this.